### PR TITLE
Preparing for the altruistic-1 testnet

### PR DESCRIPTION
**Pull request by nodersteam**
- Added path with testnets.
- Implemented a checker gentx for altruistic-1

### CODE REVIEW

**Review by bjaanes — APPROVED** [review]

File: .github/workflows/testnets.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - 'altruistic-1/gentx/**.json'
```
Comment (bjaanes) [inline .github/workflows/testnets.yml:6]: This probably needs to be 'testnets/altruistic-1/gentx/**.json'

File: .github/workflows/testnets.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: CI
```
Comment (bjaanes) [inline .github/workflows/testnets.yml:1]: Let's change the name to Gentx checker or something?

File: .github/workflows/testnets.yml (hunk 1 above)
Comment (icodragon) [reply .github/workflows/testnets.yml]: Corrected the path, thanks

File: .github/workflows/testnets.yml (hunk 2 above)
Comment (icodragon) [reply .github/workflows/testnets.yml]: I like the name gentx checker - let's use it?

**Pull request merged**